.devcontainer/devcontainer.json

```
@@ -18,5 +18,8 @@
             ]
         }
     },
+    "remoteEnv": {
+        "PATH": "${containerEnv:PATH}:/workspaces/dotnet/.dotnet"
```

Not thrilled about hard coding the path here but couldn't find a good sub to use. Tried using `${localWorkspaceDir}` but that was giving out paths in `/var`

---

in dotnet/runtime we use this:

```suggestion
        "PATH": "${containerWorkspaceFolder}/.dotnet:${containerEnv:PATH}",
```